src/prasad/MissingLink.java

```diff
@@ -0,0 +1,85 @@
+package prasad;
+
+/*Assignment - 8 : Missing_Link & Multiwindow_Handle
+
+1. Print missing link button text
+Note: use missing link form from resources folder
+
+2. Verify that by click on any button on form new window is opening
+Note: use Window_Handling Form from resources
+*/
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import com.sun.xml.bind.v2.schemagen.xmlschema.List;
+
+public class MissingLink {
+
+	public WebDriver start() {
+		System.setProperty("webdriver.chrome.driver", ".//resources/windows/chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get(
+				"C:\\Users\\Prasad Chitale\\Documents\\java_selenium\\SeleniumTechnoOct2020\\resources\\forms\\MissingLink.html");
+		driver.manage().window().maximize();
+		driver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
+
+		return driver;
+	}
+
+	void findMissing(WebDriver driver) {
+		ArrayList<WebElement> hyperLinks = new ArrayList<WebElement>();
```

Comment:
Hi @PrasadCOct2020 
you should use findElements(//a); using this you can get all the elements in list, may be your code will not for other elements.